faq/index.html

```diff
@@ -15,765 +16,891 @@
-                    <h5 class="mb-0">
-                      <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#b11" aria-expanded="true" aria-controls="b11">
-                        <span class="number">11. </span>How to begin validating?
-                      </button>
-                      <span class="sign">+</span>
-                    </h5>
-                  </div>
-
-                  <div id="b11" class="collapse multi-collapse" aria-labelledby="a11" >
-                    <div class="card-body">
-                      If you want to become our validator, your steps are following:
-                      <ol>
-                        <li>Download the node software: <a href="https://validator.parseclabs.org/">https://validator.parseclabs.org/</a></li>
-                        <li>Install and start the node. During the first start the signer-key is created. Note the signer-address.</li>
-                        <li>Wait for the node to completely sync the chain.</li>
-                        <li>Fund the signer-address with enough Ether to pay for block submission transactions.</li>
-                        <li> Pick a slot for use by the validator. Prepare enough funds on a secure wallet (cold storage or multi-sig). Deposit funds from the secure wallet to the validator slot, passing in the signer-address.</li>
-                        <li>The node software will notice the deposit and start proposing blocks within 2 to 3 epochs.</li>
-                      </ol>
-                    </div>
-                  </div>
-                </div>
-                <div class="carda">
-                  <div class="card-header" id="a12">
-                    <h5 class="mb-0">
-                      <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#b12" aria-expanded="true" aria-controls="b12">
-                        <span class="number">12. </span>What is your organization model?
-                      </button>
-                      <span class="sign">+</span>
-                    </h5>
-                  </div>
-
-                  <div id="b12" class="collapse multi-collapse" aria-labelledby="a12" >
-                    <div class="card-body">
-                      You can find our organization model <a href="https://docs.google.com/presentation/d/13-I4QqIX4B8adAnaY65Ik_H24uouVi2YTP9ZQrCAcVs/edit#slide=id.g3d321dc7a7_10_0">here</a>. If you will have some questions regarding the model - feel free to ask us in Telegram or Twitter.
-                    </div>
-                  </div>
-                </div>
-                <div class="carda">
-                  <div class="card-header" id="a13">
-                    <h5 class="mb-0">
-                      <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#b13" aria-expanded="true" aria-controls="b13">
-                        <span class="number
```

Comment:
"Latest posts" section should not be hardcoded, but generated from the actual latest posts